webui/react/src/components/UserAvatarTable.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import Avatar, { Props as AvatarProps } from 'shared/components/Avatar';
+import useUI from 'shared/contexts/stores/UI';
+import { getDisplayName, UserNameFields } from 'utils/user';
+
+import css from './UserAvatarTable.module.scss';
+
+export interface Props extends Omit<AvatarProps, 'darkLight' | 'displayName'> {
+  compact?: boolean;
+  user: UserNameFields;
+}
+
+const UserAvatarTable: React.FC<Props> = ({ user, compact, ...rest }) => {
```

Comment:
I'm not normally a stickler for names but this name is very inaccurate; this isn't a table. `UserInfo`, `UserBadge`, `UserIndicator`, there are lots of names that would be an accurate representation of what this is.

Reply:
My bad.. 😅